docs/tutorials/pytorch2onnx.md

```diff
@@ -214,6 +221,7 @@ The table below lists the models that are guaranteed to be exportable to ONNX an
 |    YOLOv3    |    `configs/yolo/yolov3_d53_mstrain-608_273e_coco.py`    |       Y       |        Y        |       |
 | Faster R-CNN |   `configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py`   |       Y       |        Y        |       |
 |  Mask R-CNN  |     `configs/mask_rcnn/mask_rcnn_r50_fpn_1x_coco.py`     |       Y       |        Y        |       |
+|     DETR     |             `configs/detr/detr_r50_8x2_150e_coco.py`     |       Y       |        N        |       |
```

Comment:
Does `DETR` in ONNX support batch inference?

Reply:
Currently it's not supported and not recommended to use batch inference in ONNX, since huge performance gap between exported onnx model and torch model (e.g. 33.5 vs 39.9 for onnxruntime and torch, with a batch size 2)